credible/pcl-quickstart.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Quickstart'
-description: 'A quickstart guide to writing assertions and using the Credible Layer CLI'
+description: 'A quickstart guide to writing assertions and using the Credible Layer CLI (reminder that is pretains to local deployments only)'
```

Comment:
Don't add disclaimers like this in the description. If you think there should be a disclaimer then add it as a "Note" box. I don't see the value in mentioning that this only pertains to local deployments.